docs/solutions/sql_server_always_on_availability_groups/modules/network/securitylist/variables.tf

```diff
@@ -0,0 +1,21 @@
+variable "proto_number_tcp" {
```

Comment:
Not blocking, but it's better to define these as locals since they are not meant to ever be changed by the user. Example from another config:
locals {
  tcp_protocol = "6"
  all_protocols = "all"
  anywhere = "0.0.0.0/0"
}

Reply:
ok...will put in refactor.